diff -urN foomatic-db-engine-3.0.1.org/configure.in foomatic-db-engine-3.0.1/configure.in
--- foomatic-db-engine-3.0.1.org/configure.in	2004-08-10 02:32:12.069058928 +0200
+++ foomatic-db-engine-3.0.1/configure.in	2004-08-10 02:32:27.552705056 +0200
@@ -18,9 +18,9 @@
 dnl LIBS="$LIBS"
 
 AM_PATH_XML2(2.0.0)
-if test ! -n "$XML_LIBS" ; then
-  AM_PATH_XML(1.8.17)
-fi
+#if test ! -n "$XML_LIBS" ; then
+#  AM_PATH_XML(1.8.17)
+#fi
 dnl CFLAGS="$CFLAGS $XML_CFLAGS"
 dnl LIBS="$LIBS $XML_LIBS"
 
diff -urN foomatic-db-engine-3.0.1.org/Makefile.in foomatic-db-engine-3.0.1/Makefile.in
--- foomatic-db-engine-3.0.1.org/Makefile.in	2004-08-10 02:32:12.065059536 +0200
+++ foomatic-db-engine-3.0.1/Makefile.in	2004-08-10 02:33:01.803498144 +0200
@@ -53,7 +53,7 @@
 ## C settings
 CC=@CC@
 CFLAGS=@CFLAGS@
-XML_CFLAGS=@XML_CFLAGS@
+XML_CFLAGS=@XML_CPPFLAGS@
 XML_LIBS=@XML_LIBS@
 
 ## Settings for Perl
